From 1ea6426a7aefcabdf2771e60a8575f6ad6f3bf71 Mon Sep 17 00:00:00 2001
From: Andrew Wong <andrew.kw.w@gmail.com>
Date: Fri, 2 Sep 2016 15:48:56 +0900
Subject: [PATCH] Explicitly specify the components needed from HDF5

In CMake 3.6, the default behavior of FindHDF5 will require more
components than Field3D needs.
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e9ad44f..1610c2e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -38,7 +38,7 @@ PROJECT ( field3d )
 set( CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${PROJECT_SOURCE_DIR}/cmake )
 
 FIND_PACKAGE (Doxygen)
-FIND_PACKAGE (HDF5)
+FIND_PACKAGE (HDF5 COMPONENTS C)
 IF ( CMAKE_HOST_WIN32 )
 # f3dinfo relies on program_options but don't include it, since
 # for some reason, unlike all the other boost components, a link is
